game/plugin/skills/runecrafting/src/runecrafting.plugin.kts

```diff
@@ -2,7 +2,6 @@ import org.apollo.game.message.impl.*
 import org.apollo.game.model.entity.EquipmentConstants
 import org.apollo.game.model.event.impl.LoginEvent
 
-private val changeAltarObjectConfigId = 491
```

Comment:
Probably should've done this change in a separate commit

Reply:
oops, missed that